### PR TITLE
Add failure injection blog post

### PR DESCRIPTION
**Pull request by adleong**
Signed-off-by: Alex Leong <alex@buoyant.io>

### CODE REVIEW

File: linkerd.io/content/blog/failure-injection-using-the-service-mesh-interface-and-linkerd.md [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+implementation.
+
+## Example
+
+We'll start by installing the Linkerd CLI and deploying it on our Kubernetes
+cluster:
+
+```bash
+> curl https://run.linkerd.io/install | sh
+> export PATH=\$PATH:\$HOME/.linkerd2/bin
```
Comment (alpeb) [inline linkerd.io/content/blog/failure-injection-using-the-service-mesh-interface-and-linkerd.md]: @alex it appears the backslashes here cause errors

Comment (adleong) [reply linkerd.io/content/blog/failure-injection-using-the-service-mesh-interface-and-linkerd.md]: good catch, thanks

**Comment by scottcarol** [comment]
Great post! The demo basically worked, I had a couple questions:

..._We should now see that the application is healthy:_

Are my RPS so much lower than your screenshot under this sentence because you're running Booksapp with more pods than I am?

Screen Shot 2019-07-18 at 4 17 03 PM

..._Not very well, it seems!_

My Booksapp was still able to open: 
`kubectl port-forward deploy/webapp 7000 & open http://localhost:7000`
Screen Shot 2019-07-18 at 4 18 47 PM

**Comment by scottcarol** [comment]
One tiny punctuation nit - everything else looks great:

"so lets remove that configured error rate:" --> **let's**

**Pull request merged**